src/app/constructor/map/map-constructor.component.ts

```diff
@@ -26,18 +26,50 @@ export class MapConstructorComponent implements OnInit {
     @ViewChild('fileInput') fileInput: ElementRef;
 
     @Input() canBeActive: boolean;
-    @Input() mapSubViewData: MapViewInfoDto;
-    @Input() default: string;
+    @Input() mapSubview: MapViewInfoCreateDto | MapViewInfoDto;
+
+    @Output() defaultSubviewChange: EventEmitter<string> = new EventEmitter<string>();
+    @Input()
+    set defaultSubview(value) {
+        this.defaultSubviewValue = value;
+        this.defaultSubviewChange.emit(value);
+    };
+    get defaultSubview(): string {
+        return this.defaultSubviewValue;
+    };
+
+    @Output() nameChange: EventEmitter<string> = new EventEmitter<string>();
```

Comment:
According to design, we have two blocks: for Contructor / Map and Contructor / Dashboard. Name and description located in Constructor / Map block. In case of moving name and description upper, we will have one big component with almost all logic in it